app/controllers/api/v1/reservations_controller.rb

```diff
@@ -1,2 +1,23 @@
 class Api::V1::ReservationsController < ApplicationController
+  def index
+    reservations = Reservation.find_by(params[:user_id])
+
```

Comment:
Good job in getting the reservation, but I suggest instead of using `find_by`, use `where` or a similar query because if you use `find_by` you'll get just the first element that matches with your condition, and in the requirements, we are supposed to get a list of user reservations.

Reply:
Thanks for pointing out 👍